client/App/pages/LandingPage/components/Dropdown/Dropdown.module.scss

```diff
@@ -0,0 +1,30 @@
+.dropdown {
+  background-color: #f6f6f6;
+  width: 100%;
+  padding: 45px 0;
+
+  &__list {
+    display: grid;
```

Comment:
This gid here is kinda redundant, but if You want to use it, go ahead :). 
2 thing  is that you can't hard coding values in css. Good practice during styling container is to use percents, or for example flex: 1. It looks nice on every screen.

Reply:
I thought about using em/rem values, but imo this solusion also has some disadvantages. I will replace hard values by some more flexible units like percents